enterprise/suppress-user/internal/badgerdb/badgerdb.go

```diff
@@ -176,7 +181,15 @@ func (b *Repository) Add(suppressions []model.Suppression, token []byte) error {
 			if suppression.Canceled {
 				err = wb.Delete([]byte(key))
 			} else {
-				err = wb.Set([]byte(key), []byte(""))
+				var value []byte
+				metadata := model.Metadata{
+					CreatedAt: suppression.CreatedAt,
+				}
+				value, err = json.Marshal(metadata)
```

Comment:
We could probably use the jsoniter library which we have been using everywhere

WDYT @atzoum